doc/sphinx/practical-tools/utilities.rst

```diff
@@ -262,6 +268,40 @@ The following makefile rules can be extended.
   One can append lines to the generated ``.merlin`` file extending this
   target.
 
+.. _coqmakefilelocallate:
+
+CoqMakefile.local-late
+++++++++++++++++++++++
+
+The optional file ``CoqMakefile.local-late`` is included at the end of the generated
+file ``CoqMakefile``.  The following is an incomplete list of accessible variables:
+
+:COQ_VERSION:
+   describes the version of ``coqc`` being used, and can be used to
+   provide different behavior depending on what version of Coq is
+   being used
+:COQMAKEFILE_VERSION:
+   describes the version of Coq that was used to generate the
+   Makefile, and can be used to detect version mismatches
+:ALLDFILES:
+   contains the list of generated dependency files, and can be used,
+   for example, to instruct ``make`` to recompute all dependencies
+   when some file changes by writing ``$(ALLDFILES): myfile`` or to
+   indicate that some file must be generated before dependencies can
+   be computed by writing ``$(ALLDFILES): | mygeneratedfile``
```

Comment:
```suggestion
   the list of generated dependency files, which can be used,
   for example, to cause ``make`` to recompute dependencies
   when one or more files change by writing ``$(ALLDFILES): myfile`` or to
   indicate that a file must be generated before dependencies can
   be computed by writing ``$(ALLDFILES): | mygeneratedfile``
```

Reply:
No reason to explicitly say "one or more"
```suggestion
   the list of generated dependency files, which can be used,
   for example, to cause ``make`` to recompute dependencies
   when files change by writing ``$(ALLDFILES): myfiles`` or to
   indicate that files must be generated before dependencies can
   be computed by writing ``$(ALLDFILES): | mygeneratedfiles``
```